public/app/plugins/datasource/cloudwatch/components/ConfigEditor.tsx

```diff
@@ -74,9 +74,9 @@ export const ConfigEditor = (props: Props) => {
       <h3 className="page-heading">CloudWatch Logs</h3>
       <div className="gf-form-group">
         <InlineField
-          label="Retry Timeout"
+          label="Query Result Timeout"
           labelWidth={28}
-          tooltip='Cloudwatch Logs allows for a maximum of 30 concurrent queries. If Grafana hits a concurrent max query error from Cloudwatch Logs it will auto-retry requesting a query for up to 30min. This retry timeout strategy is configurable. Must be a valid duration string, such as "15m" "30s" "2000ms" etc.'
+          tooltip='Grafana will poll for Cloudwatch Logs query results every second until Done status is returned from AWS or timeout is exceeded, in which case Grafana will return an error. The default period is 30 minutes. Note: For Alerting, the timeout defined in the config file will take precedence. Must be a valid duration string, such as "15m" "30s" "2000ms" etc.'
```

Comment:
I guess it can be nanoseconds, too, since we support numbers but maybe we should not include that in the official recommendation in order to align with other similar fields in Grafana. What do you think?

Reply:
Im ok keeping it like this too, I doubt nanosecond will be used anyway